src/dhtnode/node/RequestHandlers.d

```diff
@@ -38,4 +39,5 @@ static this ( )
     requests.add(RequestCode.Get, "Get", &Get.handle);
     requests.add(RequestCode.Mirror, "Mirror", &Mirror.handle);
     requests.add(RequestCode.GetAll, "GetAll", &GetAll.handle);
+    requests.add(RequestCode.GetChannels, "GetChannels", &GetChannels.handle);
```

Comment:
You don't want this bit (+ all `GetChannel` files in the node) as a separate commit?

Reply:
Well, it's required for the dhtproto tests to pass.

Reply:
So I thought I'd put it all in one commit, along with the submodule update.

Reply:
OK, that's a fair point, I've missed that.